source/guide_foundryvtt.rst

```diff
@@ -0,0 +1,221 @@
+.. highlight:: console
+
+.. author:: revilowaldow <oliver@warlow.engineer>
+.. author:: Glumbosch <glumbosch.home.blog>
+
+.. tag:: self-hosting
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/foundryvtt.png
+      :align: center
+
+############
+Foundry VTT
+############
+
+.. tag_list::
+
+
+  `Foundry Virtual Tabletop <https://foundryvtt.com/>`_ is an alternative to Roll20 and many other platforms that enable you to play tabletop role playing games online.
+  Foundry is not free software, but you can host it on your own server.
+  
+  The first step is to purchase a license from their `website <https://foundryvtt.com/>`_.
+  
+  If you pay by SEPA transfer the payment can take up to a week, faster payment methods are available.
+
+----
+
+.. note:: This guide assumes you are completely unfamiliar with the basic concepts of linux
+
+Prerequisites
+=============
+
+SSH Introduction
+------------------
+
+We assume you don’t know what SSH is. It’s a way to securely send text commands to another computer. Servers usually don’t have a graphical user interface. You control them via text commands. To send commands to Uberspace we need an ssh-client program.
+
+First visit https://dashboard.uberspace.de/dashboard/authentication, login and then set your ssh password on this page.
+
+On Windows you will need to install SSH, it is included on Mac.
+
+
+.. note:: To install OpenSSH using Windows Settings
+
+  * First open **Settings**, select **Apps > Apps & Features**, then select **Optional Features**.
+  * Check the list to see if OpenSSH is already installed.
+  * If not, at the top of the page, select Add a feature, then find **OpenSSH Client**, and click **Install**
+
+
+
+For this demo we are going to assume that your Uberspace user is called ``isabell``.
+
+On Windows open a **command prompt**, on Mac open the **Terminal** app.
+
+Enter ``ssh isabell@isabell.uber.space`` and press return.
+
+You will be prompted for a password.
+
+Type your Uberspace password and press return. 
+
+It is normal for there to be no feedback as you type.
+
+
+You are now connected to your server. Congratulations!
+
+
+
+Set the node js version
+--------------------------
+
+Foundry uses something called :manual:`Node.js <lang-nodejs>`; we need to tell our Uberspace server to use version 14.
+
+
+Paste the following command into your ssh window:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use node 14
+ Selected Node.js version 14
+ [isabell@stardust ~]$
+
+To verify the change you can enter:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '14'
+ [isabell@stardust ~]$
+
+Installation
+======================
+
+Prepare installation folders
+-----------------------------
+
+``mkdir`` is the command we will use to create two new folders **foundryvtt** and **foundrydata**. 
+
+``mkir myfolder`` creates the folder *"myfolder"*.
+
+Press enter after each line:
```

Comment:
```suggestion
Create the required directories:
```
Same as above. To keep them short, guides can assume a basic level of linux knowledge.

Reply:
As above, this guide specifically does not want to assume that level of knowledge; it seeks to _concisely_ provide only the needed information to get this use case started.

Reply:
I see your intent Luto, and having a specific concrete example for people to work from is helpful.

For people do indeed have a preference on their ssh/cli text editor, then they most likely have experience enough to interpret the commentary to use within that editor (after all preference suggests comparison and evaluation)